core/src/main/java/org/mule/runtime/core/internal/streaming/CursorManager.java

```diff
@@ -56,6 +59,16 @@ public CursorManager(MutableStreamingStatistics statistics) {
     this.statistics = statistics;
   }
 
+  @Override
+  public void start() throws MuleException {
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. ([squid:S1186](https://sonarqube.kbuild.msap.io/coding_rules#rule_key=squid%3AS1186))

[See it in SonarQube](https://sonarqube.kbuild.msap.io/project/issues?id=mule-runtime-mule&pullRequest=7643&issues=AWirZZnmotIXnmsUDp_U&open=AWirZZnmotIXnmsUDp_U)

[ISSUE_KEY]: AWirZZnmotIXnmsUDp_U